### DISCUSSION TITLE
Photoprism | Cache folder

### DISCUSSION THREAD

**Discussion by smtdev** [tteck/Proxmox#574] [Q&A] [answered]
Hey,

Is there any way to change the default cache folder? I use ZFS as filesystem and I would like to create a dataset with a different recordsize for these matters. I have had a quick glance at the source code but I did not read any value related to caching, so I suspect that this would be in the Photoprism roof. 

Photoprism docs[ have an article about cache folder with Docker](https://docs.photoprism.app/getting-started/advanced/caching/). There is also [a flag](https://docs.photoprism.app/getting-started/config-options/#storage) value to change storage paths:

- PHOTOPRISM_STORAGE_PATH | `--storage-path` | Writable storage PATH for sidecar, cache, and database files
- PHOTOPRISM_CACHE_PATH | `--cache-path` | Custom cache PATH for sessions and thumbnail files optional

Do you know how should I use this flags?

Thank you again for your help and, of course, your AMAZING work!!

**Comment by tteck** [comment, marked as answer]
Take a look at `nano /var/lib/photoprism/.env`

**Reply by smtdev** [reply]
How fast to reply!😅.

Yeah, that's the way! 

Thank you again.
